src/js/common/schemaform/reducers/index.js

```diff
@@ -91,7 +91,22 @@ export default function createSchemaFormReducer(formConfig) {
   return (state = initialState, action) => {
     switch (action.type) {
       case SET_DATA: {
-        const newState = _.set([action.page, 'data'], action.data, state);
+        // TODO: Write the documentation that says this is a thing (persistent schema
```

Comment:
While this makes sense, I'm a little hesitant to push it just before a launch, but maybe some testing would help. Is there a test that could be added to `test/common/schemaform/reducers/index.unit.spec.js` that exercises `recalculateSchemaAndData()` in the way that requires this fix to only update the data and not the schema or uiSchema?